rhwp/rhwp-studio/src/ui/agent-sidebar/effort-slider.css

```
@@ -0,0 +1,141 @@
+/* 추론 강도 슬라이더 — 두툼한 트랙 + 디텐트 점 + 스프링으로 따라오는 엄지. */
+
+.ag-eslider {
+  --ag-eslider-pad: 14px;
+  display: flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)